test/DataMCSpectraComparison/histosSimplified.py

```diff
@@ -311,7 +321,11 @@ def ntuplify(process, fill_gen_info=False):
 #    	    	#process.pathOur2016 *=obj * process.SimpleNtupler * process.SimpleNtuplerEmu
 #    	    	process.pathOur2016 *= process.SimpleNtuplerEmu
 
+<<<<<<< HEAD
+ntuplify(process) #to have ntuples also running in interactive way
```

Comment:
This is the original Alice code.
If you tell me that this is the problem again, I will comment out this.